json-handler.js

```diff
@@ -22,6 +22,104 @@ module.exports = function(path, auth) {
         }
       }
     },
+    jobStats: {
+      method: 'GET',
+      path: path + '/jobs/stats',
+      config: {
+        auth: auth,
+        handler: function(request, reply) {
+          var now = new Date();
+          var agenda = this.agenda;
+
+          async.parallel({
```

Comment:
Swapped to parallel. Originally was going to use waterfall to calculate later stats from earlier ones but abandoned that idea later on.